src/gui/guiFormSpecMenu.h

```diff
@@ -27,6 +27,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "inventorymanager.h"
 #include "modalMenu.h"
 #include "guiTable.h"
+#include "guiScrollBar.h"
```

Comment:
Use forward declaration:
- Move this line to `guiFormSpecMenu.cpp`.
- Add below there (line 36 and further) a line with `class GUIScrollBar;`.